ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -744,5 +744,4 @@ test-suite doctest
   default-language: Haskell2010
   ghc-options: -Wno-unused-packages
   build-depends:
-    base,
-    latex-svg-image,
```

Comment:
Why was this removed?

It is needed for the doctests

Reply:
I'm surprised by this, I thought the `ghc-options: -Wno-unused-packages` on the line above would take precedence over your custom options, but probably I was wrong. User options are always the ones that dominate.

So yeah, a normal build won't complain (even if the package was unused to the eyes of Cabal) but if you specify `-Wunused-packages` explicitly then it will.